spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe User do
+  describe 'validation' do
+    let(:user) { create(:user) }
+    let(:invalid_user) { build(:user, email: nil) }
+    context 'when user provided email' do
```

Comment:
instead write this 
`  it 'has valid factory' do
    expect(build(:user)).to be_valid
  end`